doc/async_operations.md

```diff
@@ -1,5 +1,24 @@
 # Asynchronous operations
 
-You are reading a draft of the next documentation and it's in continuos update so
-if you don't find what you need please refer to:
-[C++ wrapper classes for the ABI-stable C APIs for Node.js](https://nodejs.github.io/node-addon-api/)
\ No newline at end of file
+Node.js native add-ons often need to execute long running tasks and to avoid
+blocking the **event loop** they have to run them asynchronously from the
+**event loop**.
+In the Node.js model of execution there are **two threads**, the first is the
+**event loop** thread, that represents the thread where JavaScript code is
+executing. The node.js guidance is to avoid you blocking other work queued on the
+event loop by. Therefore, we need to do this work on another thread.
+
+All this means that native add-ons need to leverage async helpers from libuv as
+part of their implementation. This allows them to schedule work to be executed
+asynchronously so that their methods can return in advance of the work being
+completed.
+
+Node Addon API provides an ABI-stable interface to support functions which covers
+the most common asynchronous use cases. You have two abstract class to implement
```

Comment:
The general guidance is to not formulate sentences using "you". So, we should have "~~You have~~there are".

"abstract class**es**"